packages/styling/src/styles/zIndexes.ts

```diff
@@ -0,0 +1,8 @@
+export namespace ZIndexes {
```

Comment:
Why do we need this? Even if we have keytips using z-index, other things should not be.
Having a file like this encourages the use of z-index, which we should be doing. And it also implies that z-index is global, which it isn't. We can't control how users consume and layer OUFR components, so trying to manage z-index between them is useless.